files/imagestreams/images.yml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
```

Comment:
The rhel7 ImageStream is still required as it is utilized by the s2i golang image. I would recommend turning this object into a list so that both imagestreams can be utilized

Reply:
NP. I made it into a template like the others